vips/vip-156/vip-156.ts

```diff
@@ -120,11 +177,23 @@ export const vip131 = () => {
         params: [FEE_OUT],
       },
 
+      {
+        target: PSM_USDT,
+        signature: "setVAIMintCap(uint256)",
+        params: [VAI_MINT_CAP],
+      },
+
       {
         target: VAI_CONTROLLER_PROXY,
         signature: "setBaseRate(uint256)",
         params: [BASE_RATE_MANTISSA],
       },
+
+      {
+        target: TREASURY,
+        signature: "withdrawTreasuryBEP20(address,uint256,address)",
+        params: [USDT, USDT_FUNDING_AMOUNT, PSM_USDT],
```

Comment:
I don't think we should transfer directly the USDT from the treasury to the PSM. I would transfer the USDT from the treasury to the Normal Timelock, and then I would invoke `swapStableForVAI`, setting the VTreasury address as the `receiver` address. This way we'll have VAI in the Treasury and we'll be able to recover the "deposited" USDT